extern/boostd-data/ldb/db.go

```diff
@@ -322,7 +322,7 @@ func (db *DB) MarkIndexErrored(ctx context.Context, pieceCid cid.Cid, err error)
 	}
 
 	md.Error = err.Error()
-	md.ErrorType = fmt.Sprintf("%T", err)
+	md.ErrorType = fmt.Sprintf("%T", sourceErr)
```

Comment:
Nice catch 👍 
I think we also need to change the previous line:
md.Error = sourceErr.Error()

Reply:
You're right, good catch.